sdk/template/aztemplate/version.go

```diff
@@ -0,0 +1,15 @@
+//go:build go1.17
```

Comment:
@jhendrixMSFT Do you see concerns with generating a version.go file?  I know it would come from generater, however having it as version.go makes it very easy for customers to find this information which I believe is valuable.

Reply:
We put the version info in constants.go.  Having a version.go file for two constants is a very .NET thing to do ;)  However if it _really_ fixed stuff we could do it.

Reply:
FWIW other languages like c/c++ often create a version header file as well.